sources/packages/forms/src/form-definitions/sfaa2021-22.json

```diff
@@ -10223,9 +10223,9 @@
               "tags": [],
               "properties": {},
               "conditional": {
-                "show": null,
-                "when": null,
-                "eq": "",
+                "show": "true",
+                "when": "howWillYouBeAttendingTheProgram",
+                "eq": "Full Time",
                 "json": ""
               },
```

Comment:
I still see this question in PT application. 
![image](https://github.com/bcgov/SIMS/assets/54600590/aed020d7-66bc-468d-91e9-21dec2e01f79)

Is it not the one listed here to be removed ? 
![image](https://github.com/bcgov/SIMS/assets/54600590/c987c706-3670-4b42-bbdc-7d6a1b136741)

Reply:
Good catch. Done.